packages/slice-machine/src/legacy/lib/models/common/widgets/LinkToMedia/index.ts

```diff
@@ -41,7 +41,6 @@ export const LinkToMediaWidget: Widget<Link, typeof schema> = {
       placeholder: "",
       select: "media",
       allowText: true,
-      repeat: false,
```

Comment:
not sure why this was here but IMO it's not needed and it's misleading because there's no repeat in link to media

Reply:
I think it was because of the first version of types internal, when it was required to set `repeat`